drivers/sensor/voltage_divider/voltage.c

```diff
@@ -171,7 +179,7 @@ static int voltage_init(const struct device *dev)
 	PM_DEVICE_DT_INST_DEFINE(inst, pm_action);                                                 \
                                                                                                    \
 	SENSOR_DEVICE_DT_INST_DEFINE(inst, &voltage_init, PM_DEVICE_DT_INST_GET(inst),             \
-			      &voltage_##inst##_data, &voltage_##inst##_config, POST_KERNEL,       \
-			      CONFIG_SENSOR_INIT_PRIORITY, &voltage_api);
+				     &voltage_##inst##_data, &voltage_##inst##_config,             \
+				     POST_KERNEL, CONFIG_SENSOR_INIT_PRIORITY, &voltage_api);
```

Comment:
Non-related change, it seems.

Reply:
was done by the auto-formatter. Reverted the change